index.js

```diff
@@ -0,0 +1 @@
+console.log('hello world')
```

Comment:
This is great, but let's add a semi-colon to the end of the line to maintain our intended code style.

```suggestion
console.log('hello world');
```